website/docs/r/kubernetes_cluster_node_pool.html.markdown

```diff
@@ -147,6 +149,8 @@ The following arguments are supported:
 
 ~> **NOTE:** A route table must be configured on this Subnet.
 
+* `workload_runtime` - (Optional) Used to specify the workload runtime. Allowed values are `OCIContainer` and `WasmWasi`.
```

Comment:
It looks as though `WasmWasi` node pools are also in preview, so another note here would be good
```suggestion
* `workload_runtime` - (Optional) Used to specify the workload runtime. Allowed values are `OCIContainer` and `WasmWasi`.

~> **Note:** WebAssembly System Interface node pools are in Public Preview - more information and details on how to opt into the preview can be found in [this article](https://docs.microsoft.com/en-us/azure/aks/use-wasi-node-pools)
```